AirCasting/Graph/GraphView.swift

```diff
@@ -18,12 +20,16 @@ struct GraphView<StatsViewModelType>: View where StatsViewModelType: StatisticsC
     
     var body: some View {
         VStack(alignment: .trailing) {
-            SessionHeaderView(action: {},
-                              isExpandButtonNeeded: false,
-                              isSensorTypeNeeded: false,
-                              isCollapsed: Binding.constant(false),
-                              session: session,
-                              sessionStopperFactory: sessionStoppableFactory).padding()
+            HStack {
+                backButton
```

Comment:
I know it's now the same as Android's but it doesn't feel iOSy. 
Maybe we could ask Michael if he's ok with moving the `backButton` to the top? 
I just think it's really important to follow Apple's design principles so I'd like to try to convince Michael.

Reply:
Sure! :) Can we merge this and move it to the top in another PR?